lib/measures/default_feature_reports/measure.rb

```diff
@@ -537,8 +537,8 @@ def run(runner, user_arguments)
       
     # electricity_heating  
     electricity_heating = sql_query(runner, sql_file, "AnnualBuildingUtilityPerformanceSummary", "TableName='End Uses' AND RowName='Heating' AND ColumnName='Electricity'")
-    feature_report.reporting_periods[0].end_uses[:electricity][:heating] = convert_units(electricity_heating, 'GJ', 'kBtu')
-    
+    feature_report.reporting_periods[0].end_uses.electricity.heating = convert_units(electricity_heating, 'GJ', 'kBtu')
```

Comment:
@rawadelkontar the end_uses is no longer a hash, it is a class, so you have to use the class methods

Reply:
I fixed this first error, you will have to fix the remaining ones